ionics_fits/models/containers.py

```diff
@@ -69,28 +69,31 @@ def func(
             }
             ys.append(model.func(x, model_params))
 
-        return np.stack(ys).T
+        return np.array(ys)
 
     def estimate_parameters(
         self,
         x: Array[("num_samples",), np.float64],
-        y: Array[("num_samples", "num_y_channels"), np.float64],
+        y: Array[("num_y_channels", "num_samples"), np.float64],
         model_parameters: Dict[str, ModelParameter],
     ):
+        y = np.atleast_2d(y)
         for idx, (model_name, model) in enumerate(self.models):
             params = {
                 param_name: model_parameters[f"{model_name}_{param_name}"]
                 for param_name in model.parameters.keys()
             }
-            model.estimate_parameters(x, y[:, idx], params)
+            model.estimate_parameters(x, y[idx], params)
 
     def calculate_derived_params(
         self,
         x: Array[("num_samples",), np.float64],
-        y: Array[("num_samples",), np.float64],
+        y: Array[("num_y_channels", "num_samples"), np.float64],
         fitted_params: Dict[str, float],
         fit_uncertainties: Dict[str, float],
     ) -> Tuple[Dict[str, float], Dict[str, float]]:
+        y = np.atleast_2d(y)
```

Comment:
why is this needed?

Reply:
(isn't it already done by the fitter? or is the idea that the model may be used without the fitter?)

Reply:
I don't think it's actually needed, will take out